test/e2e/app-dir/phase-changes/cookies.test.ts

```diff
@@ -0,0 +1,149 @@
+/* eslint-env jest */
+import { nextTestSetup } from 'e2e-utils'
+import { retry } from 'next-test-utils'
+
+describe('setting cookies', () => {
+  const { next, isNextDeploy, skipped } = nextTestSetup({
+    files: __dirname,
+    // `patchFile` and reading runtime logs are not supported in a deployed environment
+    skipDeployment: true,
```

Comment:
did you mean to include this gate? I see there are a handful of `isNextDeploy` checks in the tests below

Reply:
ouch, good catch, that's what it get for copy-pasting lol